src/main/java/com/palantir/gradle/revapi/config/RevapiConfig.java

```diff
@@ -41,14 +47,43 @@ public final RevapiConfig addVersionOverride(GroupNameVersion groupNameVersion,
                 .build();
     }
 
+    public final Set<AcceptedBreak> acceptedBreaksFor(GroupNameVersion groupNameVersion) {
+        return Streams.stream(Optional.ofNullable(acceptedBreaks().get(groupNameVersion.version())))
+                .flatMap(versionedAcceptedBreaks -> versionedAcceptedBreaks.acceptedBreaks()
+                        .get(groupNameVersion.groupAndName())
+                        .stream())
+                .collect(Collectors.toSet());
+    }
+
+    public final RevapiConfig addAcceptedBreaks(
+            GroupNameVersion groupNameVersion,
+            Set<AcceptedBreak> acceptedBreaks) {
+
+        PerProjectAcceptedBreaks existingAcceptedBreaks =
+                acceptedBreaks().getOrDefault(groupNameVersion.version(), PerProjectAcceptedBreaks.empty());
+
+        PerProjectAcceptedBreaks newPerProjectAcceptedBreaks = existingAcceptedBreaks.merge(
+                groupNameVersion.groupAndName(),
+                acceptedBreaks);
+
+        Map<Version, PerProjectAcceptedBreaks> newAcceptedBreaks = new HashMap<>(acceptedBreaks());
+        newAcceptedBreaks.put(groupNameVersion.version(), newPerProjectAcceptedBreaks);
+
+        return ImmutableRevapiConfig.builder()
+                .putAllVersionOverrides(versionOverrides())
+                .putAllAcceptedBreaks(newAcceptedBreaks)
```

Comment:
generally prefer `.from(this).setAllAcceptedBreaks(newAcceptedBreaks)` so yoyu can't forget to set some new parameter.
The set also makes it clear that we are overriding rather than adding

Reply:
I agree, but that `set...` method does not seem to exist.

Reply:
Ah, it just doesn't have the `set` prefix.